zppy/templates/default.ini

```diff
@@ -189,6 +191,7 @@ reference_data_path_ts_rof = string(default="")
 # Required for "model_vs_model" "enso_diags"/"qbo"/"area_mean_time_series" runs: `reference_data_path_ts`
 # Required for "model_vs_model" "enso_diags"/"qbo"/"area_mean_time_series"/"streamflow" runs: `ts_num_years_ref`, `ts_subsection`
 # Required for "model_vs_model" "enso_diags"/"streamflow"/"tc_analysis" runs: `ref_final_yr`, `ref_start_yr`
+# Required for "model_vs_model" "tropical_subseasonal" runs: `ref_final_yr`, `ref_start_yr`, `reference_data_path_ts_daily`, `ts_num_years_ref`, `ts_subsection`
```

Comment:
I think I was doing this by listing all the sets that certain parameters are used in. E.g., it would have been:
```
# Required for "model_vs_model" "enso_diags"/"qbo"/"area_mean_time_series"/"streamflow"/"tropical_subseasonal" runs: `ts_num_years_ref`, `ts_subsection`
# Required for "model_vs_model" "enso_diags"/"streamflow"/"tc_analysis"/"tropical_subseasonal" runs: `ref_final_yr`, `ref_start_yr`
# Required for "model_vs_model" "tropical_subseasonal" runs: `reference_data_path_ts_daily`
```
But actually that's getting to be a bit unwieldy. So, the by-set method is probably better anyway. If you have time, maybe change this whole block to:
```
# Required for "model_vs_model" "area_mean_time_series" runs: `reference_data_path_ts`, `ts_num_years_ref`, `ts_subsection`
# Required for "model_vs_model" "diurnal_cycle" runs: `reference_path_climo_diurnal`
# Required for "model_vs_model" "enso_diags" runs: `ref_final_yr`, `ref_start_yr`, `reference_data_path_ts`, `ts_num_years_ref`, `ts_subsection`
# Required for "model_vs_model" "qbo" runs: `reference_data_path_ts`, `ts_num_years_ref`, `ts_subsection`
# Required for "model_vs_model" "streamflow" runs: `gauges_path`, `ref_final_yr`, `ref_start_yr`, `reference_data_path_ts_rof`,  `ts_num_years_ref`, `ts_subsection` 
# Required for "model_vs_model" "tc_analysis" runs: `ref_final_yr`, `ref_start_yr`, `reference_diurnal_path_tc`
# Required for "model_vs_model" "tropical_subseasonal" runs: `ref_final_yr`, `ref_start_yr`, `reference_data_path_ts_daily`, `ts_num_years_ref`, `ts_subsection`
```

Reply:
Thanks for the review. I agree that list by set perhaps is more naturally. I made a new commit to address this.